pandas/tests/io/json/test_readlines.py

```diff
@@ -219,3 +221,17 @@ def test_readjson_nrows_requires_lines():
     msg = "nrows can only be passed if lines=True"
     with pytest.raises(ValueError, match=msg):
         pd.read_json(jsonl, lines=False, nrows=2)
+
+
+def test_readjson_lines_chunks_fileurl(datapath):
+    # GH 27135
+    # Test reading line-format JSON from file url
+    os_path = datapath("io", "json", "data", "line_delimited.json")
+    file_url = "file://localhost" + os_path
+    if system() == "Windows":
+        file_url = PureWindowsPath(file_url)
+    path_reader = pd.read_json(os_path, lines=True, chunksize=1)
```

Comment:
This isn't what is being tested right? I think would be preferable if you construct this literally

Reply:
No it's not. I added the OS check because the Azure Pipeline checks for Windows will fail due to different file url format.

By constructing it literally, do you mean something like this?

```python
    if system() == "Windows":
        file_url = "file:\localhost" + os_path
    else:
        file_url = "file://localhost" + os_path
```

Reply:
Just using literal values - so `df_list = pd.DataFrame([[1, 2], [3, 4], [5, 6]], columns=["a", "b"])`